packages/edit-navigation/src/components/menus-editor/index.js

```diff
@@ -47,6 +53,15 @@ export default function MenusEditor( { blockEditorSettings } ) {
 				<MenuEditor
 					menuId={ menuId }
 					blockEditorSettings={ blockEditorSettings }
+					onDeleteMenu={ ( deletedMenu ) => {
+						const newStateMenus = filter( stateMenus, ( menu ) => {
```

Comment:
nit: `stateMenus.filter()` would also do the trick

Reply:
done